### ISSUE TITLE
[BUG] Schema error: Schema contains duplicate unqualified field name 'time' during query

### ISSUE THREAD

**Issue by Benxiaohai001** [cnosdb/cnosdb#945]
**Describe the bug**
Schema error: Schema contains duplicate unqualified field name 'time' during query

**To Reproduce**
```sql
CREATE DATABASE database8;
CREATE TABLE m0(f0 STRING , TAGS(t0, t1));
CREATE TABLE m5(f0 STRING , TAGS(t0, t1));
INSERT m0(TIME, t0, f0, t1) VALUES(CAST (3076611083640115437 AS TIMESTAMP), '1o3#pvr', '0.37379244615716656', '');
INSERT m5(TIME, t0, f0, t1) VALUES(CAST (6670499103009143464 AS TIMESTAMP), '5324969635426306259', 'fF㛞._|&蹂', 'Q4'), (CAST (6670499103009143464 AS TIMESTAMP), '', 'K', ''), (CAST (5857788160081912114 AS TIMESTAMP), '', '365128501014843670', 'kJTT ᴰ쥟');
SELECT ALL * FROM m0 JOIN m5 ON starts_with(((((m0.f0)||(m0.t0)))||(CAST( 1886464716 AS DOUBLE))), lower(m0.f0)) WHERE CAST( ((((7398101348861774578)-(2783618793006118078)))+(((8930985319142038558)+(6285959963762503381)))) AS BOOLEAN) UNION ALL SELECT * FROM m0 JOIN m5 ON starts_with(((((m0.f0)||(m0.t0)))||(CAST( 1886464716 AS DOUBLE))), lower(m0.f0)) WHERE NOT (CAST( ((((7398101348861774578)-(2783618793006118078)))+(((8930985319142038558)+(6285959963762503381)))) AS BOOLEAN)) UNION ALL SELECT ALL * FROM m0 JOIN m5 ON starts_with(((((m0.f0)||(m0.t0)))||(CAST( 1886464716 AS DOUBLE))), lower(m0.f0)) WHERE (CAST( ((((7398101348861774578)-(2783618793006118078)))+(((8930985319142038558)+(6285959963762503381)))) AS BOOLEAN)) IS NULL;
```
**Expected behavior**
no error

**Additional context**
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/40594937/221492709-bcdfd1ef-0d48-4f44-92ae-3ae6f4e78db7.png">

**Comment by ZuoTiJia** [comment]
Waiting for DF to fix.

**Comment by yukkit** [comment]
fixed in datafusion  v20.0.0

**Comment by yukkit** [comment]
resolved,  closed by https://github.com/cnosdb/cnosdb/issues/1096
